test.js

```diff
@@ -129,7 +139,7 @@ it('normal throw', function() {
     'throw new Error("test");'
   ], [
     'Error: test',
-    /^    at Object\.exports\.test \(.*\/line1\.js:1001:101\)$/
+    /^    at Object\.exports\.test \((?:.*\/)?line1\.js:1001:101\)$/
```

Comment:
IMHO, the tests should be more explicit: do we expect a relative or absolute path?

If relative, we should probably simply remove the `.*/` prefix…